code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -239,33 +242,6 @@ obj/item/weapon/gun/energy/staff/focus/attack_self(mob/living/user as mob)
 		return 0
 	return 1
 
-
-/obj/item/weapon/gun/energy/wand/polymorph

```

Comment:
Why did you remove it?

Reply:
Because it's part of the PR.

Reply:
You said you were keeping it for asminbuse.

Reply:
No I did not.